SConstruct

```diff
@@ -0,0 +1,3 @@
+# -*- python -*-
+from lsst.sconsUtils import scripts
+scripts.BasicSConstruct("deploy_testing_distrib")
```

Comment:
If you change this to `scripts.BasicSConstruct("deploy_testing_distrib", noCfgFile=True)` the `deploy_testing_distrib.cfg` file can be removed.  I other metapackages I checked all have a `.cfg` file but it should be unnecessary.  As a sanity check, I confirmed that this change is functional. See `sconsUtils` for more info: https://github.com/lsst/sconsUtils/blob/763ede5c87e95b8fa030e1a295e433046841a622/python/lsst/sconsUtils/scripts.py#L59-L62

We should probably do this for all of the metapackages.